recipes/edward/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "edward" %}
+{% set version = "1.3.3" %}
+{% set sha256 = "104d58321c5040235b3039ae3215c3c7881073e6aa88bb0b8ca1141ca87c4891" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    
+  run:
+    - python
+    - numpy >=1.7
+    - six >=1.10.0
+
+test:
+  imports:
+    - edward
+    - edward.criticisms
+    - edward.inferences
+    - edward.inferences.conjugacy
+    - edward.models
+    - edward.util
+
+about:
+  home: http://edwardlib.org
+  license: Apache-2.0
+  license_file: '{{ environ["edward"] }}/LICENSE.txt'
```

Comment:
The `license_file` line is incorrect. It should be:

```
license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Reply:
My bad, I thought ``RECIPE_DIR`` is a placeholder where you will replace it with your variable according to your recipe.